src/components/Icons/Icons.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SVG, PATH } from './Icons.styles';
+
+const Icon = ({ width, filled, height, icon, isPink }) => {
+  return (
+    <>
+      <SVG width={width} height={height} filled={filled}>
+        <PATH icon={icon} isPink={isPink} />
+      </SVG>
+    </>
+  );
+};
```

Comment:
```
export const Heart = () => <Icon icon="Heart" />
```
Consider having all Icons as named exports from this file, for ease of use. Or maybe index.js